build.gradle

```diff
@@ -33,19 +45,20 @@ def graphqlJavaVersion = '2021-03-22T20-02-28-3a701a0f'
 
 def releaseVersion = System.env.RELEASE_VERSION
 version = releaseVersion ? releaseVersion : getDevelopmentVersion()
-group = 'com.graphql-java'
+group = 'com.atlassian'
 
 
 repositories {
     mavenLocal()
     maven { url "http://dl.bintray.com/andimarek/graphql-java" }
```

Comment:
we should remove the bintray address as graphql java is only available on maven central going forward